src/Generator/Library.cs

```diff
@@ -175,10 +175,15 @@ public static void SetClassBindName(this ASTContext context, string className, s
                 @class.Name = name;
         }
 
-        public static void SetClassAsValueType(this ASTContext context, string className)
+        public static void SetClassAsValueType(this ASTContext context, string className, Driver driver)
```

Comment:
Please change this to only pass the information you actually need: Driver.Options.IsCLIGenerator. That is, use a Boolean and not a Driver parameter.